DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Commands/NoteCommands.cs

```diff
@@ -18,5 +19,17 @@ public NoteCommands(NoteContext notesContext)
             _notesContext.Notes.Add(noteEntity);
             await _notesContext.SaveChangesAsync();
         }
+
+        public async Task DeleteNote(int noteId)
+        {
+            var note = new Note
+            {
+                Id = noteId
+            };
+
+            _notesContext.Entry(note).State = EntityState.Deleted;
```

Comment:
Yes, it works. With EntityState I had to make it tracked first and then call method Remove.